pkg/cstor/volumereplica/v1alpha1/cstorvolumereplica.go

```diff
@@ -171,3 +174,27 @@ func (b *ListBuilder) WithFilter(pred ...Predicate) *ListBuilder {
 	b.filters = append(b.filters, pred...)
 	return b
 }
+
+// BuilderForAPIObject returns an instance of the Builder object based on cvr api object.
+func BuilderForAPIObject(cvr *apis.CStorVolumeReplica) *Builder {
```

Comment:
Instead of returning `Builder`, return CVR typed object. This will save lot of changes.

Reply:
Removed the builder code changes.